### PR TITLE
Advancedmode api changes 22 april vpn

### PR DESCRIPTION
**Pull request by sdharav**

### CODE REVIEW

File: gbpservice/nfp/configurator/agents/firewall.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 rest_timeout = [
     cfg.IntOpt(
         'rest_timeout',
-        default=60,
+        default=360,
```
Comment (dpaks) [inline gbpservice/nfp/configurator/agents/firewall.py]: Why is this modified to 360 seconds?

Comment (sdharav) [inline gbpservice/nfp/configurator/agents/firewall.py]: because in 60 VPN routes are not getting configured

**Comment by njagadish** [comment]
This PR is not needed as this is rebased to a different daily branch (27th April)
